src/CommunityToolkit.Maui.Core/Views/DrawingView/Service/DrawingViewService.tizen.cs

```diff
@@ -0,0 +1,222 @@
+﻿using Microsoft.Maui.Graphics.Skia;
+using SkiaSharp;
+
+namespace CommunityToolkit.Maui.Core.Views;
+
+/// <summary>
+/// Drawing view service
+/// </summary>
+public static class DrawingViewService
+{
+	/// <summary>
+	/// Get image stream from points
+	/// </summary>
+	/// <param name="points">Drawing points</param>
+	/// <param name="imageSize">Image size</param>
+	/// <param name="lineWidth">Line Width</param>
+	/// <param name="strokeColor">Line color</param>
+	/// <param name="background">Image background</param>
+	/// <returns>Image stream</returns>
+	public static async ValueTask<Stream> GetImageStream(IList<PointF> points, Size imageSize, float lineWidth, Color strokeColor, Paint? background)
+	{
+		var image = GetBitmapForPoints(points, lineWidth, strokeColor, background);
+
+		if (image is null)
+		{
+			return Stream.Null;
+		}
+
+		// Defer to thread pool thread https://github.com/CommunityToolkit/Maui/pull/692#pullrequestreview-1150202727
+		return await Task.Run(() =>
```

Comment:
Do we need to await the Task here? Can't we just return it?

Same for the line [50](https://github.com/CommunityToolkit/Maui/pull/692/files#diff-a26ec24614bb6020677735a034f76ab28d2279a4afd7678f7c22357888ce3c92R50)

Reply:
Oh, good point, we can just return